spec/models/power_relay_spec.rb

```diff
@@ -11,11 +11,26 @@ class SomeRelay < Relay
 RSpec.describe SomeRelay do
 
   it { is_expected.to validate_presence_of :ip }
-  it { is_expected.to validate_presence_of :port }
+  it { is_expected.to validate_presence_of :outlet }
   it { is_expected.to validate_presence_of :username }
   it { is_expected.to validate_presence_of :password }
   it { is_expected.not_to validate_presence_of :auto_logout_minutes }
 
+  it "should allow a range of 1-16 outlets" do
```

Comment:
We still have a bunch of old specs following this pattern, but just as we're switching from `should` to `expect` rspec syntax, we're trying to move away from "should" in our descriptions. So something like "it 'allows a range of 1-16'"